config/Dockerfiles/rpmbuild/Dockerfile

```diff
@@ -41,4 +41,5 @@ COPY rpmbuild-test.sh /home/rpmbuild-test.sh
 # Run the build script
 ENTRYPOINT ["bash", "/home/rpmbuild-test.sh"]
 # Run the container as follows
+# Note: For stage builds, don't do production=no, simply don't pass a production var in
```

Comment:
My intent was to just have it set so that if you don't pass in a production environment variable, you are assumed to not be production.  This way, production=yes production=true production=YES production=please all work